common/extract-examples.rb

```diff
@@ -24,6 +24,10 @@
 
 PREFIXES = {
   dc:     "http://purl.org/dc/terms/",
```

Comment:
Shouldn't 'dc' be changed?

Reply:
as mentioned on API, this is there because of backwards compatibility, as previously "dc" was used for terms, although we've changed them all to "dcterms" now.